tf2onnx/graph_matcher.py

```diff
@@ -107,16 +107,30 @@ def _to_pattern(self, pattern_or_name):
             return pattern_or_name
 
         if isinstance(pattern_or_name, six.text_type):
-            return self._name_to_pattern[pattern_or_name]
+            return self._name_to_pattern.get(pattern_or_name, None)
```

Comment:
returning None is the default do you can remove the ', None'

Reply:
Thanks for your comment! I have removed the ',None' to make the expression clearer.